### PR TITLE
HAFI

### PR DESCRIPTION
**Pull request by Harkerfield**

### CODE REVIEW

**Comment by JacobTheEldest** [comment]
Dat commit history

File: client/src/Reserve/DualTimeSelector.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
-function DualTimeSelector() {
+function DualTimeSelector({ timeList, setTimeList }) {
   const [startTime, setStartTime] = useState('09:00');
   const [endTime, setEndTime] = useState('10:00');
-  const [timeList, setTimeList] = useState([]);
+  // const [timeList, setTimeList] = useState([]);
   const [name, setName] = useState(`VUL-0`);
 
+  useEffect(() => {
```
Comment (JacobTheEldest) [inline client/src/Reserve/DualTimeSelector.js:9]: This new useEffect looks like it'd be an infinite loop. Does it cause that recursive re-render problem?

File: client/src/Reserve/Reserve.js
Comment (JacobTheEldest) [inline client/src/Reserve/Reserve.js:1]: This is a yuge file. Maybe we can put an item on the backlog to refactor into subcomponents at some point.

File: package-lock.json
Comment (JacobTheEldest) [inline package-lock.json:1]: Installing stuff to the root directory was probably unintentional. Need to delete this file.

File: package.json
Comment (JacobTheEldest) [inline package.json:1]: Delete this too.

**Review by JacobTheEldest — CHANGES_REQUESTED** [review]
A few adjustments needed, please.

**Comment by JacobTheEldest** [comment]
> > Dat commit history
> 
> Baby steps...

Squashing commits might suit your workflow better.

File: client/src/Reserve/UserForm.js
Comment (JasonMartin85) [inline client/src/Reserve/UserForm.js:1]: You have the old squadron names hard coded here… we have new squadron names now..

**Review by hackek — APPROVED** [review]
Approved!

**Review by JacobTheEldest — APPROVED** [review]

**Pull request merged**